### PR TITLE
Np ultima cramonly test

### PR DESCRIPTION
**Pull request by nikellepetrillo**

### CODE REVIEW

**Comment by nikellepetrillo** [comment]
Link to a cram only test: https://cromwell.gotc-dev.broadinstitute.org/api/workflows/1/2f44f222-3913-4c51-97db-a7049eb5d368/metadata

**Review by jessicaway — APPROVED** [review]

File: verification/VerifyUltimaGenomicsWholeGenomeCramOnly.wdl [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+    File test_cram
+    File truth_cram
+    File test_crai
+    File truth_crai
+
+    Boolean? done
+  }
+
+  ## TEMP - Can't use this simple call, because the picard version used by UltimaGenomicsWholeGenomeGermline is based on an old version of Picard,
```
[the remark this answers is not in the document]
Comment (meganshand) [reply verification/VerifyUltimaGenomicsWholeGenomeCramOnly.wdl:20]: It should be ok to update to the latest version of Picard for all of the Ultima pipelines at this point.

File: verification/test-wdls/TestUltimaGenomicsWholeGenomeCramOnly.wdl [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+                                    [ # File outputs
+                                    UltimaGenomicsWholeGenomeCramOnly.agg_quality_distribution_pdf,
+                                    UltimaGenomicsWholeGenomeCramOnly.agg_gc_bias_pdf,
+                                    UltimaGenomicsWholeGenomeCramOnly.selfSM,
+                                    UltimaGenomicsWholeGenomeCramOnly.output_cram_md5,
+                                    UltimaGenomicsWholeGenomeCramOnly.output_cram_index,
+                                    UltimaGenomicsWholeGenomeCramOnly.output_cram,
+                                    ],
+                                    # File? outputs
+                                    select_all([UltimaGenomicsWholeGenomeCramOnly.output_bam_index]),
```
Comment (meganshand) [inline verification/test-wdls/TestUltimaGenomicsWholeGenomeCramOnly.wdl:60]: Hopefully in all cases for `UltimaGenomicsWholeGenomeCramOnly` these output bam and output bam_index files should always be null. Not sure if that's worth adding a test for?

**Review by meganshand — APPROVED** [review]
One small question, but otherwise looks good to me. Thanks!

File: verification/test-wdls/TestUltimaGenomicsWholeGenomeCramOnly.wdl (hunk 2 above)
Comment (nikellepetrillo) [reply verification/test-wdls/TestUltimaGenomicsWholeGenomeCramOnly.wdl:60]: @meganshand if these are always null i can just remove them from this pipeline_outputs array. Do you want me to add a check somewhere to see if the bams are null and fail if they are not null?

File: verification/VerifyUltimaGenomicsWholeGenomeCramOnly.wdl (hunk 1 above)
[the remark this answers is not in the document]
Comment (nikellepetrillo) [reply verification/VerifyUltimaGenomicsWholeGenomeCramOnly.wdl:20]: ok great! we will update to the latest version of picard in a future pr 👍

**Comment by nikellepetrillo** [comment]
retest this please

**Comment by nikellepetrillo** [comment]
retest this please

File: verification/test-wdls/TestUltimaGenomicsWholeGenomeCramOnly.wdl (hunk 2 above)
Comment (nikellepetrillo) [reply verification/test-wdls/TestUltimaGenomicsWholeGenomeCramOnly.wdl:60]: Hi @meganshand, just making sure you saw the comment above!

**Comment by nikellepetrillo** [comment]
retest this please

**Pull request merged**